index.rst

```diff
@@ -12,18 +12,633 @@
+Another conclusion from this plot is that the number of diaObjects in Pre-convolution is higher, but after filtering it ends up being lower (by a ~300 diaObjects margin) than the other flavors. Our plot also shows that a significant portion of these could be in the bin of 20 or more diaSources, which is interesting.
+
+In the following figure we have a scatter plot of these diaObjects on sky coordinates.
+
+.. figure:: /_static/figures/good_diaObj_zoomed_sky.png
+   :name: good_diaObj_zoomed_sky
+   :target: ../_images/good_diaObj_zoomed_sky.png
+   :alt: Scatter of diaObjects in the sky
+
+   Scatter plot showing the position of DIA source associations for each filter and each DIA flavor or mode. Size of the points is proportional to the number of associated diaSources.
+
+We observe the spatial distribution of the diaObjects and their number of diaSources also displayed as the size of the scatter points. We can observe that the objects are clustered around what could be bright sources in the field. In contrast to the default mode we see that there are less points in the pre-convolution mode, although the difference is subtle.
+
+We can understand that the associated diaSources in this plot should have already a significant cut, and they are mostly equivalent.
+
+
+
+
+Fake source injection analysis
+==============================
+
+Number of matches
+-----------------
+
+We can expand the table that we built before to include the number of fake source matches.
+
+.. _table-nfakematches:
+
+.. table:: Table with number of fake matches per DIA flavor.
+
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    |           | Good diaSrc | Good diaObject | N Fakes Match | contamination |  Matches w/flag cuts|
+    +===========+=============+================+===============+===============+=====================+
+    |   Default |       13244 |           6166 |        4627.0 |       50172.0 |              2391.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    |  Pre-Conv |       10909 |           6381 |        4594.0 |       55649.0 |              1732.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    | Auto Mode |       13727 |           6698 |        4321.0 |       45728.0 |              2348.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+
+We can see that the amount of matches decreases to almost a 50% when applying the flag cuts. Our conclusion is that maybe our set of flags is too agressive, and it is not really discriminating the real transients from the potential artifacts, but instead vettoing on information that might or might not be relevant.
```

Comment:
This is alarming! We should investigate this further since it suggests we are losing ~50% of our injected fakes from the flag cuts, which meant to be very conservative.

Reply:
Definitively, I think this wasn't expected, at least by me.